e2e-tests/upgrade/run

```diff
@@ -28,6 +28,11 @@ IMAGE_BACKUP='percona/percona-server-mongodb-operator:1.1.0-backup'
 
 if [[ "${MONGO_VER}" == "4.2" ]]; then
     IMAGE_MONGOD="percona/percona-server-mongodb-operator:1.1.0-mongod4.0"
+elif [[ "${MONGO_VER}" == "4.4" ]]; then
+    API='psmdb.percona.com/v1-4-0'
+    IMAGE='percona/percona-server-mongodb-operator:1.4.0'
+    IMAGE_MONGOD="percona/percona-server-mongodb-operator:1.4.0-mongod4.2"
+    IMAGE_BACKUP='percona/percona-server-mongodb-operator:1.4.0-backup'
```

Comment:
let us test all versions since 1.4.0
I prefer to avoid adding `statefulset_upgrade-rs0-4.4.yml`

Reply:
done